perf_dashboard/helpers/download.py

```diff
@@ -80,15 +80,15 @@ def download_benchmark_csv(days):
     for i in range(len(cur_release_names)):
         cur_release = cur_release_names[i]
         sub_str = cur_release[len(current_release) + 1:].split("-")[0]
-        cur_release_dates[i] = [0] * 3
-        cur_release_dates[i] = [sub_str[0:4], sub_str[4:6], sub_str[6:8]]
+        cur_release_dates[i] = [0] * 4
+        cur_release_dates[i] = [sub_str[0:4], sub_str[4:6], sub_str[6:8], sub_str]
 
     master_release_dates = [[]] * len(master_release_names)
     for i in range(len(master_release_names)):
         master_release = master_release_names[i]
         sub_str = master_release[len("master") + 1:].split("-")[0]
-        master_release_dates[i] = [0] * 3
-        master_release_dates[i] = [sub_str[0:4], sub_str[4:6], sub_str[6:8]]
+        master_release_dates[i] = [0] * 4
```

Comment:
I have no clue what this does, should we make this more clear or add comments maybe?

Reply:
This just parsing the release with SHA name. Sure, I will add a common or make it interpretable.